jetty-core/jetty-session/src/main/java/org/eclipse/jetty/session/AbstractSessionManager.java

```diff
@@ -933,16 +935,28 @@ public void setUsingCookies(boolean usingCookies)
      * @return whether the session management is handled via URLs.
      */
     @Override
+    public boolean isUsingUriParameters()
+    {
+        return _usingUriParameters;
+    }
+
+    public void setUsingUriParameters(boolean usingUriParameters)
+    {
+        _usingUriParameters = usingUriParameters;
+    }
+
+    @Deprecated
     public boolean isUsingURLs()
     {
-        return _usingURLs;
+        return isUsingUriParameters();
     }
 
+    @Deprecated
```

Comment:
```suggestion
    /**
     * @deprecated use {@link #setUsingUriParameters(boolean)} instead, will be removed in Jetty 12.1.0
     */
    @Deprecated(since = "12.0.1", forRemoval=true)
```